pkg/sawmill/sawmill.go

```diff
@@ -0,0 +1,87 @@
+package sawmill
+
+// If you're wondering why this package is named sawmill, I do too: https://clips.twitch.tv/PlacidOutstandingPelicanCharlieBitMe
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"cloud.google.com/go/logging"
+	"google.golang.org/api/option"
+	logpb "google.golang.org/genproto/googleapis/logging/v2"
+)
+
+// Logger holds the newly generated error logging client, so that it can be used for our own, more simple custom logging functions instead of the ones the Google provides
+type Logger struct {
+	Logger *logging.Logger // The Logger
+
+	InitError bool // If their was an error with the initialization, it won't log to the cloud
+
+	ServiceName string // The service that logging was created for
+}
+
+// InitClient creates a logging client
+func InitClient(projectID string, credentials string, environment string, serviceName string) (Logger, error) {
+	// Checks if were in the development environment, because if we are Cloud Logging is not needed
+	if environment == "DEV" {
+		return Logger{nil, false, serviceName}, nil
+	}
```

Comment:
This is definitely a good thought, but I had a different idea for how this should work. When we do the actual log event thats when we should check for the environment. 

Add a check like:

```golang
if environment != "DEV" {
    // Log to Google cloud
}

log.Println("Log the thing.")
```

Reply:
I think that it might be annoying though to have initialization happen while in DEV, because if the developer hasn't set the credentials ENV, which they probably haven't because they don't need to, InitClient will throw an error every time a serverless function is run.

Reply:
While you do have a point think about our scenario here:

1. We implement sawmill everywhere
2. I am developing locally
3. As I'm debugging, I now don't see any local log statements unless I add them in every function

In my mind, Sawmill should be our only logging platform. We shouldn't have to create duplicate logs statemens.

Reply:
Ok, makes sense